store/src/main/java/org/apache/rocketmq/store/ha/autoswitch/AutoSwitchHAService.java

```diff
@@ -309,6 +310,8 @@ public void maybeExpandInSyncStateSet(final Long slaveBrokerId, final long slave
         if (slaveMaxOffset >= confirmOffset) {
             final EpochEntry currentLeaderEpoch = this.epochCache.lastEntry();
             if (slaveMaxOffset >= currentLeaderEpoch.getStartOffset()) {
+                LOGGER.info("The slave {} has caught up, slaveMaxOffset: {}, confirmOffset: {}, epoch: {}, startOffset: {}.",
```

Comment:
How about renaming startOffset to leader epoch startOffset?

Reply:
Sure.